packages/core-data/src/fetch/__experimental-fetch-link-suggestions.js

```diff
@@ -153,6 +153,27 @@ const fetchLinkSuggestions = async (
 		);
 	}
 
+	if ( ! type || type === 'media' ) {
```

Comment:
Why do we use `! type` everywhere here?

Reply:
So it's a filter. If you don't pass `type` then it will query all the available types. However, if you pass a `type` then it will filter by a given type only (e.g. `post`, `taxonomy`...etc). See tests for examples.